src/components/shared/Drawer.jsx

```diff
@@ -0,0 +1,84 @@
+import Link from 'next/link';
+import { FaSearch } from 'react-icons/fa';
+import './Header.css';
+
+const Drawer = ({ setOpenDrawer, user, openDrawer }) => {
+  return (
+    <div
+      className={`slide ${openDrawer ? 'open' : ''} min-h-[100vh] w-[70vw]  overflow-hidden bg-california-50 p-4 text-sushi-700 lg:w-[30vw]`}
+    >
+      <div className='flex justify-end'>
+        <button onClick={() => setOpenDrawer(false)}>X</button>
+      </div>
+
+      {user && (
+        <div className='mb-4 flex items-center gap-3 border-b pb-4'>
+          <img
+            src='https://media.newyorker.com/photos/644ab20d87ae684cabca21f2/master/pass/knight-Erling-Haaland.jpg'
+            alt=''
+            className='h-12 w-12 rounded-full'
+          />
+
+          <div className='flex flex-col'>
+            <span>Hello, Sayed </span>
+            <span className='text-sm text-sushi-600'>93, 2/B, Panchlaish</span>
+          </div>
+        </div>
+      )}
+
+      {user ? (
+        <div className='mt-6 flex flex-col'>
+          <Link
+            className='rounded-md px-2 py-[2px] transition-all hover:bg-sushi-300/50 lg:py-1'
+            href='/'
+          >
+            Profile
+          </Link>
```

Comment:
Every link you use `py-[2px]` is called an arbitrary value. When tailwind does not give class for specific px then we use this arbitrary value, but for padding top and bottom 2px tailwind gives this class `py-0.5`. So we can replace using `py-0.5`. It's my preference, nevertheless, it's totally up to you.